internal/cli/up.go

```diff
@@ -168,6 +169,14 @@ func provideLogActions() store.LogActionsFlag {
 	return store.LogActionsFlag(logActionsFlag)
 }
 
+func provideKubectlLogLevel() k8s.KubectlLogLevel {
+	if klogLevel > 0 {
+		return k8s.KubectlLogLevel(logger.InfoLvl)
```

Comment:
it's worth pointing out that the --klog flag is supposed to map to klog -v, see
https://github.com/windmilleng/tilt/blob/master/internal/cli/klog.go#L29
so it might make sense to map this -v in the kubectl call as well

Reply:
(i.e., `tilt --klog 8` is supposed to be equivalent to `kubectl -v 8`)

Reply:
oh, nice!